src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -252,7 +252,7 @@ def load_arguments(self, _):
             c.argument('php_version', help='The version used to run your web app if using PHP, e.g., 5.5, 5.6, 7.0')
             c.argument('python_version', help='The version used to run your web app if using Python, e.g., 2.7, 3.4')
             c.argument('net_framework_version', help="The version used to run your web app if using .NET Framework, e.g., 'v4.0' for .NET 4.6 and 'v3.0' for .NET 3.5")
-            c.argument('linux_fx_version', help="The runtime stack used for your linux-based webapp, e.g., \"RUBY|2.3\", \"NODE|6.6\", \"PHP|5.6\", \"DOTNETCORE|1.1.0\". See https://aka.ms/linux-stacks for more info.")
+            c.argument('linux_fx_version', help="The runtime stack used for your linux-based webapp, e.g., \"RUBY|2.5.5\", \"NODE|10.14\", \"PHP|7.2\", \"DOTNETCORE|2.1\". See https://aka.ms/linux-stacks for more info.")
```

Comment:
@btardif can you confirm if we still need link to the aka.ms link we have in the help here for Linux runtimes

Reply:
Keep this link for now, I'm going to try and update it to point somewhere else.

Reply:
@panchagnula  I was able to take ownership of the link and it's now pointing to: https://github.com/Azure/app-service-linux-docs/tree/master/Runtime_Support 

so yes, lets keep this.